src/osd/PGLog.h

```diff
@@ -476,18 +476,14 @@ struct PGLog : DoutPrefixProvider {
     }
 
     void index(pg_log_dup_t& e) {
-      if (PGLOG_INDEXED_DUPS) {
```

Comment:
@tchaikov I thought of doing the same as you suggested, but then I found that `if (PGLOG_INDEXED_DUPS)` will always be `true`.

Let's hear suggestion from @ivancich.

Thanks.